engine/src/main/java/org/camunda/bpm/engine/telemetry/Internals.java

```diff
@@ -49,6 +50,20 @@ public interface Internals {
    */
   public LicenseKeyData getLicenseKey();
 
+  /**
+   * The date when the engine started to collect dynamic data, such as command executions
+   * and metrics. If telemetry sending is enabled, dynamic data resets on sending the data
+   * to Camunda.
+   *
+   * This method returns a date that is initially set to the current time on engine startup
+   * and reset on sending telemetry data to Camunda. It represents the start of the time
```

Comment:
Should the Javadocs reflect that the start date and collected data are reset on the engine startup?

Reply:
@tasso94, I think it is clear from the javadocs that the start date is reset on engine startup. Let me know if you disagree. It might only be obvious to me since I built it.

The behavior for the collected data is described in more detail on the `getMetrics()` and `getCommands()` methods. Do you suggest to include this info here as well?

Reply:
For me, it would become more apparent if it were written as follows:
> This method returns a date reset to the current time on engine startup or on sending telemetry data to Camunda.

But maybe that's just me.